corehq/apps/app_manager/templates/app_manager/partials/define_case_type_modal.html

```diff
@@ -54,9 +54,10 @@ <h4>
             >
               <option value=""></option>
             </select>
-            <span class="help-block" id="new_case_type_help">
+            <div class="help-block" id="new_case_type_error"></div>
+            <div class="help-block" id="new_case_type_help">
```

Comment:
Optional feedback: I think you could leave the help displayed at all times. The error message should be noticeable enough. That'd simplify the code and arguably the UX (it can be strange for users to have something appearing and disappearing).

Reply:
I feel the appearance is not clean when having two blocks, different color, under the input box. User won't feel something appear and disappear, there will always be a line under dropdown, and it is either a help text or different error messages.